### PR TITLE
Shadowling Balances #1

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
balance: Shadowlings glare now deals stamina damage and stuttering based on distance. It still stuns at melee range.
tweak: Shadowling glare, annihilate, and hypnosis all are now click abilities.
/:cl:

Also, alot of stuff was moved from shadowling clothing to the species itself.

### CODE REVIEW

**Comment by ghost** [comment]
Yeah I think this is actually a good change.
+1

**Comment by Pyko1** [comment]
conflicts, needs some fixing

**Comment by Pyko1** [comment]
needs more head input

File: hippiestation/code/game/gamemodes/shadowling/shadowling.dm [hunk 1]
```diff
@@ -267,7 +270,9 @@ Made by Xhuis
 	name = "Shadowling"
 	id = "shadowling"
 	say_mod = "chitters"
-	species_traits = list(NOBREATH,NOBLOOD,RADIMMUNE,NOGUNS) //Can't use guns due to muzzle flash
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NO_UNDERWEAR)
```
Comment (GuyonBroadway) [inline hippiestation/code/game/gamemodes/shadowling/shadowling.dm:273]: "NO_UNDERWEAR"

Lewd.

Comment (PopNotes) [reply hippiestation/code/game/gamemodes/shadowling/shadowling.dm]: do not fuck the sling

Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling.dm:273]: Well, Xhuis said slings don't have dicks, so it's fine.

Comment (PopNotes) [reply hippiestation/code/game/gamemodes/shadowling/shadowling.dm]: _when was this exactly_

Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling.dm:273]: a few weeks ago, over IRC

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 2]
```diff
@@ -11,47 +11,92 @@
...
+
+/obj/effect/proc_holder/spell/targeted/sling/InterceptClickOn(mob/living/caller, params, atom/t)
+	if(!ishuman(t))
+		to_chat(caller, "<span class='warning'>You can't use this ability on that!</span>")
+		revert_cast()
+		return
+	if(!ishuman(t))
+		to_chat(caller, "<span class='warning'>You may only use this ability on humans!</span>")
+		revert_cast()
+		return
```
Comment (GuyonBroadway) [inline hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: Why do you use this same check twice?

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 3]
```diff
@@ -11,47 +11,92 @@
...
+
+/obj/effect/proc_holder/spell/targeted/sling/start_recharge()
+	. = ..()
+	if(action)
+		action.UpdateButtonIcon()
+
+/obj/effect/proc_holder/spell/targeted/sling/glare //Stuns and mutes a human target for 10 seconds
 	name = "Glare"
-	desc = "Stuns and mutes a target for a decent duration."
+	desc = "Causes damage to the target's motor and speech abilities."
```
Comment (GuyonBroadway) [inline hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: Cuases damage might imply it actually harms the targets. I'd change this to "Disrupts"

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]
Aside from these two code is good.

File: hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm (hunk 3 above)
Comment (ghost) [reply hippiestation/code/game/gamemodes/shadowling/shadowling_abilities.dm]: Done

**Review by GuyonBroadway — APPROVED** [review]

**Comment by GuyonBroadway** [comment]
Changes approved, needs headmin +1's to merge.

**Comment by bardofrage** [comment]
+1

**Pull request merged**